### ISSUE TITLE
Can we run release libs compiled with emsdk  on chrome dev tools?

### ISSUE THREAD

**Issue by arpitapanda05** [emscripten-core/emscripten#14566]

**Comment by sbc100** [comment]
Yes, the libraries that are included with emsdk (which are the same as the libraries that emscripten automatically builds on demand BTW) include debug symbols so they should be debuggable in dev tools.   

As will release libraries in general, debuggability might be limited because of the optimizations done, but it should work with that caveat.
